.github/workflows/deploy.yml

```diff
@@ -0,0 +1,40 @@
+name: Deploy Java Web App to Azure
+
+on:
+  push:
+    branches: [ master, feature/deploy-to-azure ]
+
+env:
+  AZURE_WEBAPP_NAME: ${{ secrets.AZURE_WEBAPP_NAME }}
+  AZURE_WEBAPP_PUBLISH_PROFILE: ${{ secrets.AZURE_WEBAPP_PUBLISH_PROFILE }}
+  AZURE_WEBAPP_PACKAGE_PATH: ${{ github.workspace }}
+  JAVA_VERSION: '1.8'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        mongodb-version: [4.2]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Start MongoDB
+        uses: supercharge/mongodb-github-action@1.1.0
+        with:
+          mongodb-version: ${{ matrix.mongodb-version }}
+      - name: Build with Maven
+        run: mvn -B clean package jacoco:report coveralls:report -Dquarkus.package.uber-jar=true --file pom.xml
+        env:
+          COVERALLS_TOKEN: ${{ secrets.coverallsToken }}
+          SONAR_TOKEN: ${{ secrets.sonarToken }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: 'Deploy to Azure WebApp'
```

Comment:
Brauchen wir evtl noch den step zum umbenennen in app.jar?

Reply:
Nein, das ist nicht nötig